package.json

```diff
@@ -27,7 +27,9 @@
   "license": "MIT",
   "node-red": {
     "nodes": {
-      "ttn": "ttn/ttn.js"
+      "ttn-app": "ttn-app/ttn-app.js",
+      "ttn": "ttn/ttn.js",
```

Comment:
Since we change the API anyway, I'd suggest we rename `ttn` to `ttn-message` (also considering #9)